uniref-rest/src/main/java/org/uniprot/api/uniref/request/UniRefIdsDownloadRequest.java

```diff
@@ -0,0 +1,45 @@
+package org.uniprot.api.uniref.request;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import lombok.Data;
+
+import org.uniprot.api.rest.request.IdsSearchRequest;
+import org.uniprot.api.rest.validation.ValidDownloadByIdsRequest;
+import org.uniprot.store.config.UniProtDataType;
+
+/**
+ * @author sahmad
+ * @created 28/07/2021
+ */
+@Data
+@ValidDownloadByIdsRequest(accessions = "ids", uniProtDataType = UniProtDataType.UNIREF)
+public class UniRefIdsDownloadRequest implements IdsSearchRequest {
```

Comment:
Use Post in the name instead of Download, right?

Reply:
Agree